batch-submission/README.md

```diff
@@ -0,0 +1,76 @@
+# Launching the Illumina-DRAGEN job with native batch submission
+
+You can submit Illumina DRAGEN pipelines with single JSON file into Google Batch. Provided you have Batch API enabled, GCP IAM permissions and Google Cloud Secrets setup completed.
+
+It is simple to use single JSON file to config. 
+- Update the json file provided with the correct GCP project and secret information:`batchsubmit.json`
+
+```
+{
+  "taskGroups": [
+    {
+      "taskSpec": {
+        "runnables": [
+              {
+                "environment": {
+                  "secretVariables": {
+                    "JARVICE_API_USER": "projects/service-hpc-project2/secrets/jarviceApiUsername/versions/latest",
+                    "JARVICE_API_KEY": "projects/service-hpc-project2/secrets/jarviceApiKey/versions/latest",
+                    "S3_ACCESS_KEY": "projects/service-hpc-project2/secrets/batchS3AccessKey/versions/latest",
+                    "S3_SECRET_KEY": "projects/service-hpc-project2/secrets/batchS3SecretKey/versions/latest",
+                    "ILLUMINA_LIC_SERVER": "projects/service-hpc-project2/secrets/illuminaLicServer/versions/latest"
+              }
+            },          
+	   "container": {
+              "imageUri": "us-docker.pkg.dev/jarvice/images/jarvice-dragen-service:1.0-rc.5",
+              "entrypoint": "/usr/local/bin/entrypoint",
+              "commands": [
+		"--api-host", "https://illumina.nimbix.net/api",
+		"--machine", "nx1",
+		"--dragen-app", "illumina-dragen_4_2_4n",
+		"--google-sa", "883052455576-compute@developer.gserviceaccount.com",
```

Comment:
Should the Google service account be provided by the user?

Reply:
I updated the sa account to GCPPROJECTNUMBER-compute@developer.gserviceaccount.com . Yes, It has to be the user Google service account.